changelog/3994.feature.rst

```diff
@@ -0,0 +1 @@
+Added `_localfilelist` method in `~sunpy.util.scraper.Scraper` to scrap local data archives.
```

Comment:
This isn't a user facing change, so not sure it needs to be a changelog. Is this fixing a bug? If so I would mention the bug that has been fixed here instead.

Reply:
It feels like a mix of the two.